src/Raven.Server/ServerWide/Maintenance/ClusterObserver.cs

```diff
@@ -849,12 +859,14 @@ public void AddState(DatabaseObservationState state)
 
                 if (_isShardedState == false)
                     throw new InvalidOperationException($"The database {state.Name} is sharded (shard: {shardNumber}), but was initialized as a regular one.");
-
+                
                 States[shardNumber] = state;
             }
 
-            public readonly Dictionary<int, DatabaseObservationState> States;
-            public readonly RawDatabaseRecord RawDatabase;
+            public static MergedDatabaseObservationState GetEmpty()
```

Comment:
where is that used?

Reply:
To fix some existing tests that used the old function signature with the one state. Also couldn't use the static empty one.